package com.gla.Constructors_Instance.Access_Modifiers;

public class Book {
    public String ISBN;
    protected String title;
    private String author;

    public void setAuthor(String a) {
        author = a;
    }

    public String getAuthor() {
        return author;
    }
}

class EBook extends Book {

    void display() {
        System.out.println("ISBN: " + ISBN);
        System.out.println("Title: " + title);
        System.out.println("Author: " + getAuthor());
    }

    public static void main(String[] args) {

        EBook ebook = new EBook();
        ebook.ISBN = "12345";
        ebook.title = "Java Programming";
        ebook.setAuthor("James");

        ebook.display();
    }
}
